roles/sshd/defaults/main.yml

```diff
@@ -11,7 +11,7 @@ sshd_server_key_bits: "768"
 sshd_syslog_facility: "AUTH"
 sshd_log_level: "INFO"
 sshd_login_grace_time: "120"
-sshd_permit_root_login: "no"
+sshd_permit_root_login: "yes"
```

Comment:
We could remove quotes (except for yes/no values). Or we could leave them, figuring this is kind of an external role: [nickjj/ansible-sshd](https://github.com/nickjj/ansible-sshd)